### DISCUSSION TITLE
Is there way write code and did not get these strings in logs?

### DISCUSSION THREAD

**Discussion by JeanPaulLucien** [ACRA/acra#1143] [General]
```
INFORMATION,dalvikvm,Yggdrasil,Could not find method android.media.MediaCodecList.<init>, referenced from method org.acra.collector.MediaCodecListCollector.collectMediaCodecList	
WARNING,dalvikvm,Yggdrasil,VFY: unable to resolve direct method 498: Landroid/media/MediaCodecList	.<init> (I)V
```
Without changes in system configs.

**Comment by mikehardy** [comment]
This method was added in API 16. I suppose if you must run on android devices earlier than API16 then you should not initiallize ACRA with the media code list support? Alternatively I suppose you could do a PR here (if the code is not already handling it?) where you check the SDK version of the device and avoid accessing that class 

This is an incompletely specified problem though, I'm making all sorts of assumptions (about the API level you support as a minimum, how you are initializing ACRA etc) and that's an inefficient way to go about interacting with others in an open source repo - https://stackoverflow.com/help/how-to-ask - interaction is best done by reading the code, thinking about your use case, proposing a change that you have tested + experimented with, along with some discussion of the details of your use case and why your proposed change is 1) going to work well for you 2) will not negatively impact others or the structure of the code you propose to modify

**Reply by JeanPaulLucien** [reply]
> where you check the SDK version of the device and avoid accessing that class
I apologized this way. I'm not Android developer. Just #1140 was closed and I appologized that no solutions. But you answered there is. So #1140 is valid report.

**Reply by mikehardy** [reply]
1140 is not a valid report. In order to interact correctly with this repository you need to be an android developer

**Comment by JeanPaulLucien** [comment]
I get Warnings and Errors in app where developers use Acra. Here: 
https://github.com/ACRA/acra/issues/1140

I had experience with 2 apps with crash on the start. Many strings (Info, Warnings, set Info + Warn + Debug) in logs was generated by Acra. I just want to understand is it your bugs or other developers.

**Reply by mikehardy** [reply]
I feel as though you answered really quickly without truly considering what it means to specify a use case and the exact problem, that is https://stackoverflow.com/help/how-to-ask

Your comments are too vague for me to speculate further, apologies.

**Reply by JeanPaulLucien** [reply]
Just typed in same time.
